src/schema/artist/index.ts

```diff
@@ -72,7 +72,7 @@ const artistArtworkArrayLength = (artist, filter) => {
 export const ArtistType = new GraphQLObjectType({
   name: "Artist",
   interfaces: [NodeInterface],
-  fields: () => {
+  fields: (): any => {
```

Comment:
hah, yeah, I've tried a few times with this one - also, I've never thought of doing the any assignment using this syntax, good work. it feels more obvious (than a `as any` at the bottom)

Reply:
Yah felt it was more important to get everything converted rather than fuss, we can always iterate 😄